keyboards/svalboard/keymaps/features/achordion.h

```diff
@@ -57,6 +57,7 @@
 extern "C" {
 #endif
 
+  
```

Comment:
Is this accidental or just missed from the upstream?  Probably want to minimize the diff if it's unnecessary, unless your formatter is yelling that much.